app/spdb/build.sbt

```diff
@@ -250,12 +250,12 @@ def dnavarro(version: Version) = AppConfig(
     "edu.gemini.auxfile.fits.dest"               -> "/gemsoft/var/data/ictd/test/GS@SEMESTER@/@PROG_ID@",
     "edu.gemini.auxfile.fits.host"               -> "gsconfig.gemini.edu",
     "edu.gemini.auxfile.other.dest"              -> "/gemsoft/var/data/finder/GSqueue/Finders-Test/@SEMESTER@/@PROG_ID@",
-    "edu.gemini.auxfile.root"                    -> "/Users/jnavarro/.auxfile",
+    "edu.gemini.auxfile.root"                    -> "/Users/dnavarro/.auxfile",
```

Comment:
Sorry, I checked this a bit more in detail. The "dnavarro" from my code comes indeed from that commit, but it was changed at some point and I accidentally reverted it during the merge. It should be change back to "jnavarro"